packages/sdk/src/utils/compareQuotes.ts

```diff
@@ -0,0 +1,10 @@
+import type { Execute } from '../types/index.js'
+
+export const compareQuotes = (estimatedQuote: Execute, newQuote: Execute) => {
+  const estimatedIsSplit = (estimatedQuote.breakdown ?? []).length > 1
+  const newIsSplit = (newQuote.breakdown ?? []).length > 1
+
+  if (estimatedIsSplit != newIsSplit && newIsSplit) {
```

Comment:
What is the desired behavior if the expected quote goes from split route to single route? Do we still want to throw an error?